net/BUILD.gn

```diff
@@ -3311,16 +3290,149 @@ test("net_unittests") {
     }
   }
 
-  if (trial_comparison_cert_verifier_supported) {
-    # sources += [ "cert/trial_comparison_cert_verifier_unittest.cc" ]
+  if (trial_comparison_cert_verifier_supported && !is_starboard) {
+    sources += [ "cert/trial_comparison_cert_verifier_unittest.cc" ]
   }
 
   # Include transport_security_state_generator tests.
-  if (host_toolchain == current_toolchain) {
+  if (host_toolchain == current_toolchain && !use_cobalt_customizations) {
     deps += [
-      # "//net/tools/huffman_trie:huffman_trie_generator_test_sources",
-      # "//net/tools/transport_security_state_generator:transport_security_state_generator_test_sources",
+      "//net/tools/huffman_trie:huffman_trie_generator_test_sources",
+      "//net/tools/transport_security_state_generator:transport_security_state_generator_test_sources",
+    ]
+  }
+
+  if (use_cobalt_customizations) {
```

Comment:
As we are using ``use_cobalt_customizations`` to disable the tests, later on how can we distinguish which tests have been looked into and which haven't?

Reply:
I was thinking we could just have comments (i.e. see the TODO comments directly below), but we could also introduce a gn variable use_hacky_cobalt_changes (or something similar) like the c macro we have

Reply:
Prefer to use use_hacky_cobalt_changes to gate the tests need to be fixed. It's more straight forward what's the work remaining for MVP.